### ISSUE TITLE
Support for another PREFIX var

### ISSUE THREAD

**Issue by remiphilippe** [bazelbuild/rules_foreign_cc#413]
Hello,
I'm trying to build a dependency for my app, it's working fine, but the library doesn't use PREFIX but DESTDIR (https://github.com/creytiv/re/blob/master/Makefile). So `make install` always defaults to `/usr/local`.

I couldn't find a way to replace this variable, is there a parameter I missed? Or does that mean creating a new toolchain?

Thanks!

**Comment by remiphilippe** [comment]
Going to partially answer myself, I've got it to work with a custom `make_commands` : `make install DESTDIR=${INSTALLDIR}`.
For some reason though, can't seem to make it work in `make_env_vars` with the following:
```
make_env_vars = {
        "SYSROOT": "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr",
        "CC": "clang",
        "DESTDIR": "${INSTALLDIR}",
    },
```

**Comment by jsharpe** [comment]
Can you now make this work using the new env attribute merged in #489?
